port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -92,21 +93,26 @@ async def _search_as_bool(self, data: dict[str, Any], pattern: str) -> bool:
         )
 
     async def _search_as_object(
-        self, data: dict[str, Any], obj: dict[str, Any]
+        self,
+        data: dict[str, Any],
+        obj: dict[str, Any],
+        misconfigurations: dict[str, str] = {},
```

Comment:
https://www.pullrequest.com/blog/python-pitfalls-the-perils-of-using-lists-and-dicts-as-default-arguments/